lib/puppet/type/keycloak_client.rb

```diff
@@ -158,6 +158,10 @@ def insync?(is)
     desc 'rootUrl'
   end
 
+  newproperty(:backchannel_logout_url) do
+    desc 'backchannel.logout.url'
+  end 
+        
```

Comment:
```suggestion
  end

```

The validation tests are failing because of trailing white space

Reply:
fixed, thank you